app/config/publications/overview-table-columns.js

```diff
@@ -146,6 +148,6 @@ export default [
     translationKey: 'publications-table-status',
     translationKeySmall: 'publications-table-status-small',
     sortKey: 'status.position,publication-status-change.started-at',
```

Comment:
Niet echt in scope, maar zou `'status.position,-publication-status-change.started-at'` (zie `-` halverwege) gebruiksvriendelijker zijn? :thinking:

Reply:
Misschien wel, maar durf het toch niet goed aanpassen binnen deze PR, want is echt een change in functionaliteit dan.